Added error handling if organizations request is failed (#7063)
opencv/cvat: changelog.d/20231026_141231_sekachev.bs_additional_org_receive_check.md

### Fixed

> - There is not a switcher to personal workspace if an organizations request failed

Reviewer: ```suggestion
- There is no switcher to personal workspace if an organization request failed
```
